src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -367,6 +367,24 @@ def get_dependent_list(self, package_name):
         self.composite_logger.log_debug(str(len(packages)) + " dependent updates were found for package '" + package_name + "'.")
         return packages
 
+    def include_dependencies(self, packages):
```

Comment:
[optional] include_dependencies and get_dependencies_list almost does same work except for whether or not parent package is included. See, if you can refactor this.

Reply:
Same for other package manager changes as wlell

Reply:
Done for all package managers